brd-ios/breadwallet/src/ViewControllers/SecurityCenter/UpdatePinViewController.swift

```diff
@@ -404,6 +402,19 @@ class UpdatePinViewController: UIViewController, Subscriber {
             }
         }
     }
+    
+    func presentResetPinSuccess(newPin: String) {
```

Comment:
can this be delegated to the modal presenter?

Reply:
I think it can't because of dismiss completion on resetPinSuccess view